### PR TITLE
Fixes related to multichain support

### PR DESCRIPTION
**Pull request by Mi-Lan**
In this pr 2 issues were addressed
1) Top pairs data on main page was showing zero for everything because of issue with subgrpah query that was made
On base arbitrum support pr:
Screenshot 2021-09-09 at 14 52 22
In this pr fix:
Screenshot 2021-09-09 at 14 54 09
2) Another issue I found was in pair specific data graph when you select pair data, but only on networks other then mainnet. The issue was that subgraph was hardcoded into subgraph query we were making so I had to add variable so it supports multiple chains.
On arbitrum support pr:
Screenshot 2021-09-09 at 14 57 41
steps to reporduce:
1) Go to main page
2)Navigate down to Top Pairs and click on any pair that has data
3)click on third or forth chart selector
Screenshot 2021-09-09 at 14 58 42
4)Graph won't show data
On this pr after fix:
Screenshot 2021-09-09 at 14 51 34

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -1,37 +1,43 @@
...
   [SupportedNetwork.MAINNET]: ChainId[SupportedNetwork.MAINNET],
   [SupportedNetwork.XDAI]: ChainId[SupportedNetwork.XDAI],
-  [SupportedNetwork.ARBITRUM_ONE]: ChainId[SupportedNetwork.ARBITRUM_ONE],
+  [SupportedNetwork.ARBITRUM]: ChainId[SupportedNetwork.ARBITRUM],
+});
+
+export const NETWORK_SUBGRAPH_URLS = Object.freeze({
+  MAINNET: "luzzif/swapr-mainnet-alpha",
+  XDAI: "luzzif/swapr-xdai",
+  ARBITRUM: "luzzif/swapr-arbitrum-one",
```
Comment (luzzif) [inline src/constants/index.js:28]: We need to change this in the next PR, which will feature a new subgraph.

File: src/constants/index.js [hunk 2]
```diff
@@ -1,37 +1,43 @@
 export const SupportedNetwork = Object.freeze({
   MAINNET: "Mainnet",
   XDAI: "xDAI",
-  ARBITRUM_ONE: "Arbitrum",
+  ARBITRUM: "Arbitrum",
```
Comment (luzzif) [inline src/constants/index.js:4]: Wondering why this got changed to only Arbitrum

**Review by luzzif — CHANGES_REQUESTED** [review]

File: src/constants/index.js (hunk 1 above)
Comment (Mi-Lan) [reply src/constants/index.js:28]: yeah I can add issue for it once we decide to do that

**Review by luzzif — APPROVED** [review]

File: src/constants/index.js (hunk 2 above)
Comment (Mi-Lan) [reply src/constants/index.js]: you are right I changed it back there was no need for that. I just needed to structure subgraph urls object and now it works!

**Pull request merged**